userservice/src/main/java/ro/devdepot/services/UserService.java

```diff
@@ -1,9 +1,20 @@
 package ro.devdepot.services;
 
 import ro.devdepot.model.User;
+import ro.devdepot.model.UserRole;
 import ro.devdepot.model.dto.LoginRequest;
 import ro.devdepot.model.dto.LoginResponse;
 
+import java.util.List;
+
 public interface UserService {
     public LoginResponse login(LoginRequest loginRequest);
+
+    User createUser(User user);
+    User updateUser(User user, Long id);
```

Comment:
Try to create an UpdateUserDto class to use for create method, make sure to exclude fields like, password, user role